Sources/Redis/Application.Redis+configuration.swift

```diff
@@ -0,0 +1,19 @@
+import Vapor
+
+extension Application.Redis {
+    /// Configure Redis connection information.
+    /// In `configure.swift`, write `app.redis.configuration = RedisConfiguration(...)`
+    /// Or for alternate Redis IDs, `app.redis(.myRedisId).configuration = RedisConfiguration(...)`
+    /// when `extension RedisID { static let myRedisID = RedisID(...) }` has been defined.
+    public var configuration: RedisConfiguration? {
+        get {
+            self.application.redisStorage.configuration(for: self.redisID)
+        }
+        nonmutating set {
+            guard let newConfig = newValue else {
+                return
```

Comment:
Why are we now allowing configurations to be edited after they've already been set?

Reply:
I was trying to match what I saw from reading the code on `master`, which only initializes once. 

On master and on my branch:
Once the connection pools are initialized for the event loops, then changing the configuration afterwards isn't support. 

Typically, redis or other databases are setup in `configure.swift`, and then left alone. 

I'm not against someone implementing support for changing configuration after launch, but it felt out of scope for this PR.

Reply:
I suppose I worded this wrong - this used to be a `fatalError` and now it's just a silent no-op. Are the other Vapor libraries also doing a silent no-op, or are they still `fatalError`?

Reply:
oh whoops! I meant to `fatalError` there, will fix.